packages/components/src/Card/colors.css

```diff
@@ -14,14 +14,6 @@
   --card--accent-color: var(--color-invoice);
 }
 
-.outlineColor {
-  --card--accent-color: var(--color-outline);
-}
-
-.errorColor {
-  --card--accent-color: var(--color-error);
-}
-
```

Comment:
This could be considered a breaking change. I have checked Jobber proper, I dont think either of these are used

Reply:
Yep, checked that first 😄